### PR TITLE
[FVM] Deduct fees on all networks by default

### PR DESCRIPTION
**Pull request by SaveTheRbtz**
This diff makes localnet and benchnet perf tests more realistic by supporting fee deduction.  (Fee deduction currently accounts for ~30% of transactions' execution time.)

Issue: https://github.com/dapperlabs/flow-go/issues/6310

### CODE REVIEW

File: cmd/scaffold.go [hunk 1]
```diff
@@ -858,7 +858,7 @@ func (fnb *FlowNodeBuilder) initFvmOptions() {
 		fvm.WithBlocks(blockFinder),
 		fvm.WithAccountStorageLimit(true),
 	}
-	if fnb.RootChainID == flow.Testnet || fnb.RootChainID == flow.Stagingnet || fnb.RootChainID == flow.Mainnet {
+	if fnb.RootChainID == flow.Testnet || fnb.RootChainID == flow.Stagingnet || fnb.RootChainID == flow.Localnet || fnb.RootChainID == flow.Benchnet || fnb.RootChainID == flow.Mainnet {
```
Comment (pattyshack) [inline cmd/scaffold.go:861]: we can just remove this all together / enable fees everywhere?

Comment (SaveTheRbtz) [reply cmd/scaffold.go:861]: @ramtinms @janezpodhostnik @j1010001 how do you feel about this suggestion?  Or should I just negate the condition to something like `!= Emulator`?

**Review by janezpodhostnik — APPROVED** [review]
I'm ok with this, but this might cause some scenarios to break for people working with benchnet and localnet, so we should probably announce this.

File: cmd/scaffold.go (hunk 1 above)
Comment (janezpodhostnik) [reply cmd/scaffold.go:861]: There is actually a task for enabling this by default in the emulator: https://github.com/onflow/flow-cli/pull/581

Maybe it is time to just enable them everywhere by default...

**Review by pattyshack — APPROVED** [review]

**Comment by SaveTheRbtz** [comment]
bors merge

**Pull request merged**